amplicon-pooling.ipynb

```diff
@@ -523,9 +523,20 @@
    "execution_count": 6,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "seq_type = 'ITS'\n",
```

Comment:
Can you make 16S the default value and re-run the notebooks? It is most likely that this notebook will be used for 16S.